src/metrics/code_review_involvement/index.ts

```diff
@@ -19,18 +21,21 @@ export const collectCodeReviewInvolvementMetrics = async (
   const repo = payload.repository.name;
   const owner = payload.repository.owner.login;
   const pr_id = payload.pull_request.id;
+  const merged = payload.pull_request.merged;
   const created_at = moment.utc(payload.pull_request.created_at).valueOf();
   const updated_at = moment.utc(payload.pull_request.updated_at).valueOf();
   const closed_at = moment.utc(payload.pull_request.closed_at).valueOf();
   const merged_at = moment.utc(payload.pull_request.merged_at).valueOf();
```

Comment:
This should also check for `merged` - currently the output is NaN as seen in the snapshot diff, which is probably not a great fallback. Maybe `null`? That could also be a fallback for the `created_to_merged_duration` and `has_been_merged_by_author fields`

Reply:
(The comment is for `merged_at` - with GitHub highlighting only `merged` this seems confusing to me now)

Reply:
Good point. I updated the fields that depend on `merged`.